Packs/DSPM/Integrations/DSPM/DSPM.py

```diff
@@ -0,0 +1,903 @@
+    policySeverityEqual = args.get("policySeverityEqual", "")
+    validate_parameter(
+        "policySeverity", policySeverityIn, policySeverityEqual, SUPPORTED_POLICY_SEVERITIES
+    )
+
+    # check supported category type
+    categoryTypeIn = args.get("categoryTypeIn", "")
+    categoryTypeEqual = args.get("categoryTypeEqual", "")
+    validate_parameter(
+        "categoryType", categoryTypeIn, categoryTypeEqual, SUPPORTED_CATEGORY_TYPES
+    )
+
+    # check supported category type
+    statusIn = args.get("statusIn", "")
+    statusEqual = args.get("statusEqual", "")
+    validate_parameter(
+        "status", statusIn, statusEqual, SUPPORTED_STATUSES
+    )
+
+    # Check supported sorting order
+    sort_order = args.get("sort")
+    if sort_order:
+        pattern = r"^.*,(ASC|DESC)$"
+        matches = re.findall(pattern, sort_order, re.IGNORECASE)  # type: ignore
+        if not matches:
+            raise ValueError(f'This "{sort_order}" sorting order is not supported')
+
+    params = {
+        "detectionTime.equals": args.get("detectionTimeEquals"),
+        "detectionTime.greaterThanOrEqual": args.get("detectionTimeGreaterThanOrEqual"),
+        "detectionTime.greaterThan": args.get("detectionTimeGreaterThan"),
+        "detectionTime.lessThanOrEqual": args.get("detectionTimeLessThanOrEqual"),
+        "detectionTime.lessThan": args.get("detectionTimeLessThan"),
+        "policyName.in": args.get("policyNameIn"),
+        "policyName.equals": args.get("policyNameEquals"),
+        "assetName.in": args.get("assetNameIn"),
+        "assetName.equals": args.get("assetNameEquals"),
+        "cloudProvider.in": args.get("cloudProviderIn"),
+        "cloudProvider.equals": args.get("cloudProviderEquals"),
+        "destinationProjectVendorName.in": args.get("destinationProjectVendorNameIn"),
+        "destinationProjectVendorName.equals": args.get("destinationProjectVendorNameEquals"),
+        "cloudEnvironment.in": args.get("cloudEnvironmentIn"),
+        "cloudEnvironment.equals": args.get("cloudEnvironmentEquals"),
+        "policySeverity.in": args.get("policySeverityIn"),
+        "policySeverity.equals": args.get("policySeverityEquals"),
+        "categoryType.in": args.get("categoryTypeIn"),
+        "categoryType.equals": args.get("categoryTypeEquals"),
+        "status.in": args.get("statusIn"),
+        "status.equals": args.get("statusEquals"),
+        "sort": args.get("sort"),
+        "page": page,
+        "size": MAX_PAGE_SIZE
+    }
+
+    # Remove None values from params
+    params = {k: v for k, v in params.items() if v is not None}
+
+    alerts_list = client.get_alerts_list(params)
+    return alerts_list
+
+
+def get_integration_config():
+
+    integration_config = {
+        "dspmApiKey": demisto.params().get("dspmApiKey", {}).get("password"),
+        "slackMsgLifetime": demisto.params().get("slackMsgLifetime"),
+        "defaultSlackUser": demisto.params().get("defaultSlackUser"),
```

Comment:
you used **slackMsgLifetime** and **defaultSlackUser** only in this command, please change them to be the arguments for **dspm-get-integration-config** command and remove it from the params